### DISCUSSION TITLE
Spark Submit task in Prefect

### DISCUSSION THREAD

**Discussion by Sinha-Ujjawal** [PrefectHQ/prefect#4012] [Q&A]
Is there a built in task for submitting Spark jobs?

**Comment by joshmeek** [comment]
Hi @Sinha-Ujjawal there are not currently any community contributed tasks to the Task Library for Spark specific tasks (there are [Databricks](https://docs.prefect.io/api/latest/tasks/databricks.html) tasks though). If this is something you would like to see in the library we welcome and encourage contributions!
